pegasus/src/social_metadata.rb

```diff
@@ -33,46 +34,40 @@ def get_social_metadata_for_page(request)
   social_tags = {
     "code.org" => {
       "soon-hoc" => {
-        title: hoc_s(:social_hoc2018_what_create),
-        description: hoc_s(:social_hoc2018_every_student),
-        image: images[:creativity],
-        video: videos[:creativity_is]
+        title: hoc_s(:hoc2019_header),
+        description: hoc_s(:social_hoc2019_every_sudent),
+        image: images[:hoc_2019_social]
       },
       "soon-hoc-mc" => {
-        title: hoc_s(:social_hoc2018_what_create),
-        description: hoc_s(:social_hoc2018_every_student),
-        image: images[:creativity],
-        video: videos[:creativity_is]
+        title: hoc_s(:hoc2019_header),
+        description: hoc_s(:social_hoc2019_every_sudent),
+        image: images[:hoc_2019_social]
       },
       "soon-hoc-dance" => {
-        title: hoc_s(:social_hoc2018_what_create),
-        description: hoc_s(:social_hoc2018_every_student),
-        image: images[:creativity],
-        video: videos[:creativity_is]
+        title: hoc_s(:hoc2019_header),
```

Comment:
Are we doing the multi-phase launch like last year even though we aren't doing mc this year?  I guess my question is do we need soon-hoc-mc and soon-hoc-dance or will just have soon-hoc and immediately go to actual-hoc this year?

Reply:
It seems likely there will be one or more of these theme-specific variants happening, but they'll probably get refreshed assets in here before they go live.